pkg/mediationcontainer/client_websocket_transport.go

```diff
@@ -212,16 +230,19 @@ func (clientTransport *ClientWebSocketTransport) performWebSocketConnection() er
 	glog.Infof("[performWebSocketConnection]: %s", vmtServerUrl)
 
 	for !clientTransport.closeRequested { // only set when CloseTransportPoint() is called
```

Comment:
Also, logic wise, shouldn't we check:
for ! clientTransport.status != Ready {
//Reconnecting logic
}